website/docs/guides/managed_service_identity.html.markdown

```diff
@@ -71,12 +71,20 @@ Terraform can be configured to use managed identity for authentication in one of
 
 ### Configuring with environment variables
 
-Setting the `ARM_USE_MSI` environment variable to `true` tells Terraform to use a managed identity. In addition to a properly-configured management identity, Terraform needs to know the subscription ID and tenant ID to identify the full context for the Azure provider.
+Setting the `ARM_USE_MSI` environment variable to `true` tells Terraform to use a managed identity.
```

Comment:
we should also highlight the provider block field here, as we're doing it below

Reply:
Given this section talks about configuring with environment variables, I still mainly talks about env var, but added the provider block argument alternatives after each env var.